sites/skeleton.dev/src/routes/(inner)/utilities/modals/+page.svelte

```diff
@@ -325,7 +350,8 @@ new Promise<boolean>((resolve) => {
 				<h2 class="h2">Component Modals</h2>
 				<span class="badge variant-filled-warning">Advanced</span>
 			</div>
-			<p>To create custom modals, generate a new component, then pass this to the Modal system.</p>
+			<p>Skeleton allows you to generate custom modals using Svelte components. See several examples provided below.</p>
```

Comment:
> See several examples provided below.

I'm not sure if I like this sentence, but

> Check out the examples provided below

isn't better.

Reply:
During the ViteConf talk they had a great segment on docs - one of the key takeaways is less is more. I'll nix it. The examples should be visible and obvious without written direction.